pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/definition/question/FileQuestionDef.java

```diff
@@ -1,12 +1,27 @@
 package org.broadinstitute.ddp.model.activity.definition.question;
 
+import static org.broadinstitute.ddp.constants.ConfigFile.FileUploads.MAX_FILE_SIZE_BYTES;
+
+import java.util.ArrayList;
 import java.util.List;
+import javax.validation.constraints.Positive;
 
+import com.google.gson.annotations.SerializedName;
+import org.broadinstitute.ddp.interfaces.FileUploadSettings;
 import org.broadinstitute.ddp.model.activity.definition.template.Template;
 import org.broadinstitute.ddp.model.activity.definition.validation.RuleDef;
 import org.broadinstitute.ddp.model.activity.types.QuestionType;
+import org.broadinstitute.ddp.util.ConfigManager;
+
+public final class FileQuestionDef extends QuestionDef implements FileUploadSettings {
+
+    @Positive
+    @SerializedName("maxFileSize")
+    private long maxFileSize;
+
+    @SerializedName("mimeTypes")
+    private List<String> mimeTypes;
```

Comment:
Should we make this a set? What happens if someone specifies the same mime type twice?
```suggestion
    private Set<String> mimeTypes = new HashSet<>();
```

Reply:
Fixed

Reply:
Btw, may be to validate the uniqueness in the FileQuestionDef constructor and throw an exception in case of duplicates? So, it could be a list in such case..